Storage/Rust/RustRemoteTabs.swift

```diff
@@ -176,7 +176,7 @@ public class RustRemoteTabs {
     public func getClientGUIDs(completion: @escaping (Set<GUID>?, Error?) -> Void) {
         self.getAll().upon { result in
             guard result.failureValue == nil else {
-                completion(nil, result.failureValue as! String)
+                completion(nil, result.failureValue!)
```

Comment:
`⚠️nit:` same here, the guard is fine but maybe an if let works best. See previous example